homeassistant/components/knx/cover.py

```diff
@@ -88,6 +91,11 @@ def is_closing(self):
         """Return if the cover is closing or not."""
         return self._device.is_closing()
 
+    @property
+    def assumed_state(self) -> bool:
+        """Return True if unable to access real state of the entity."""
+        return self._device.current_position() is None
+
```

Comment:
The (yet) part of your proposed change will break things like like google assistant. This property is not expected to change during the life of an entity. Don't set this if this is just an temporary lack of information.

Reply:
Oh. Thats a pitty. I'll remove it again.
(We calculate the position from movement time if there is no position state from KNX. This allowed us to start in an undefined state. Without it HA always assumes the cover is closed and the frontend open-button is deactivated.)

Reply:
If you don't know the state right now, `return None` or use restore state mechanism.

Reply:
👍 xknx 0.15.4 returns `None`

Reply:
Just to clarify.  It is the state() property of the entity that should return None. Not the assumed_state() property it should always be true or false.
But maybe that is what you meant?

Reply:
Yes, that's exactly what I meant. 
I've removed the assumed_state property again so this is just the default implementation of Entity again.